extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamInfoItem.java

```diff
@@ -40,7 +42,7 @@ public class StreamInfoItem extends InfoItem {
     @Nullable
     private DateWrapper uploadDate;
     private long viewCount = -1;
-    private long duration = -1;
+    private Duration duration = Duration.ZERO;
```

Comment:
Should the default value be `0` or `null`? There are valid points for both, e.g. null safety, a valid duration being > 0. ... The default value should be documented in the corresponding getters and setters.

Reply:
I think 0 is fine. We have been using 0 as an integer to indicate "no duration available", so the same can do for Duration, too.